### PR TITLE
Content / Fundamentals V1

### PR DESCRIPTION
**Pull request by SallyMcGrath**
- moved most of the content across
- fleshed out the product pages
- created a load of default blocks ALL of which could be polished up
- created a PD placeholder block that can be replaced with links to their platform
- hooked up the backlog 

- not done yet:
- facilitation notes as I will extend the notes partial to have tabs and that's a feature not content
- read through this with fresh eyes and made sure it makes sense? I kept discovering features and building those.... Lost my sense of it really

### CODE REVIEW

**Comment by SallyMcGrath** [comment]
I ffwd as there were fixes and features from main this content depends on

File: content/fundamentals/sprints/1/prep/index.md [hunk 1]
```diff
@@ -1,11 +1,81 @@
...
 backlog_filter= 'Week 1'
 +++
 
+{{<note type="warning" title="Important">}}
+Before you come to class, make sure you've done the [precourse prep](../../../prep/) and this prep.
+{{</note>}}
 
+## Professional Accounts (2 hours)
+
+Download and install the following software or register for the apps.
```
Comment (Dedekind561) [inline content/fundamentals/sprints/1/prep/index.md:18]:
```suggestion
Below is a list of important applications that you'll use on the course.
Download and install the following software or register for the apps.
```

**Comment by Dedekind561** [comment]
Hey there, sorry only just getting round to a review for this.
This mostly looks fine to go in - I spotted a couple of things in the UI though.

## UI 
- When I start a video the play controls appear at first and then seem to disappear ( this is on the build preview )
- Some objectives aren't being rendered properly
image

I suggest merging this ASAP so I can compose the blocks for the day plans in JS1

**Comment by Dedekind561** [comment]
Sorry just noticed your top comment that you've not finished this yet - but still think we should merge this in at any rate.

**Pull request merged**